tests/system/action/poll/test_update.py

```diff
@@ -130,3 +131,231 @@ def test_not_allowed_for_non_analog(self) -> None:
             "Following options are not allowed in this state and type: "
             "votesvalid, votesinvalid, votescast"
         ) in response.data.decode()
+
+
+class UpdatePollOs3TestCase(BaseActionTestCase):
+    """
+    Tests updating polls of assignments.
```

Comment:
too specific. Maybe add a note, that this tests are for *created* polls, there are different rules for updating polls in other states.

Reply:
Removed.